### PR TITLE
Creating minimal package.json file

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

**Comment by bwinton** [comment]
Seems fine, but why do we need one? Are we planning on pushing any of this to npm? Are we going to use a bunch of third party modules?

(Also, I think the author field isn't formatted quite right.)

**Comment by mikeconley** [comment]
I'm also unsure what this is for... what are some use cases?

**Comment by gvwilson** [comment]
Added comment to #32: we may want to use packages like `file` and `path` early on. We can always install them globally, but I'm told that's poor practice.

**Comment by bwinton** [comment]
http://package-json-validator.com/ might be a good thing to run this through before merging…

**Comment by gvwilson** [comment]
Now passes with warning about `author` being missing (but `contributors` is present).

**Comment by bwinton** [comment]
What about "Software Tools in Javascript" as the author? Or "Software Tools Collective" or something?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "type": "git",
+    "url": "git+https://github.com/software-tools-in-javascript/software-tools-in-javascript.git"
+  },
+  "keywords": [
+    "education",
+    "JavaScript"
+  ],
+  "contributors": [
+    {"name": "Mike Conley", "email": "mike.d.conley@gmail.com", "url": ""},
+    {"name": "Blake Winton", "email": "bwinton@latte.ca", "url": ""},
```
Comment (bwinton) [inline package.json]: https://bwinton.latte.ca/

**Pull request merged**